log

```diff
@@ -0,0 +1,142 @@
+hal/source/sys_selftest.c: In function 'fmcECCcheck':
```

Comment:
If this is the most up-to-date output when you run `make`, then you'll have to fix all the warnings/errors after line 100.

Reply:
Also, typically, this file wouldn't be included in the PR. However, it's actually a good idea for the onboarding challenge.